src/Websites/Websites.Test/ScenarioTests/WebAppSlotTests.ps1

```diff
@@ -131,37 +131,37 @@ function Test-StartStopRestartWebAppSlot
 		$slot = $slot | Stop-AzWebAppSlot
 
 		Assert-AreEqual "Stopped" $slot.State
-		$ping = PingWebApp $slot
+		#$ping = PingWebApp $slot
```

Comment:
@Nking92 nit: for these lines you're commenting out, would you mind just removing them?